src/Apps/Artist/Routes/AuctionResults/AuctionResultsFilterContext.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useContext, useReducer } from "react"
+
+export interface AuctionResultsFilters {
+  organizations?: string[]
+  openedItemIndex: number | null
+  page?: number
+  sort?: string
+}
+
+interface AuctionResultsFiltersState extends AuctionResultsFilters {
+  reset?: boolean
+}
+/**
+ * Initial filter state
+ */
+export const initialAuctionResultsFilterState: AuctionResultsFilters = {
+  organizations: [],
+  openedItemIndex: null,
+  page: 1,
+  sort: "DATE_DESC",
+}
+
+export interface AuctionResultsFilterContextProps {
+  filters?: AuctionResultsFilters
+  onChange?: (filterState) => void
+  resetFilters: () => void
+  setFilter: (name: keyof AuctionResultsFilters, value: any) => void
+  unsetFilter: (name: keyof AuctionResultsFilters) => void
+  onAuctionResultClick?: (index: number) => void
+}
+
+/**
+ * Context behavior shared globally across the AuctionResultsFilter component tree
+ */
+export const AuctionResultsFilterContext = React.createContext<
+  AuctionResultsFilterContextProps
+>({
+  filters: initialAuctionResultsFilterState,
+  setFilter: null,
+  resetFilters: null,
+  unsetFilter: null,
+  onAuctionResultClick: null,
+})
+
+export type SharedAuctionResultsFilterContextProps = Pick<
+  AuctionResultsFilterContextProps,
+  "filters"
+  // | "onFilterClick"
+> & {
+  onChange?: (filterState) => void
+}
+
+export const AuctionResultsFilterContextProvider: React.FC<SharedAuctionResultsFilterContextProps & {
+  children: React.ReactNode
+}> = ({
+  children,
+  // counts = {},
+  filters = {},
+  // onChange,
+  // onFilterClick,
```

Comment:
If these aren't used can we clean them up?

Reply:
Yeah looking at it again the `onChange` and `onFilterClick` probably are not going to be needed in future because this `AuctionResultsFilterContext` is not going to be used in multiple pages. Going to ✂️